tide_media.install

```diff
@@ -635,3 +646,14 @@ function tide_media_update_8039() {
     }
   }
 }
+
+/**
+ * Ensure that all sites have video transcripts properly configured.
+ */
+function tide_media_update_8040() {
```

Comment:
We already have a hook to do this one, not sure if we need to add the same hook again - https://github.com/dpc-sdp/tide_media/blob/e2998cc75d0615daaeedfe4aa996e71a9a108d1f/tide_media.install#L370
I guess no need to duplicate it.